packages/flutter_tools/lib/src/commands/assemble.dart

```diff
@@ -53,6 +53,8 @@ const List<Target> _kDefaultTargets = <Target>[
   DebugIosApplicationBundle(),
   ProfileIosApplicationBundle(),
   ReleaseIosApplicationBundle(),
+  // Windows
+  DebugBundleWindowsAssets(),
```

Comment:
Is there any kind of structure to this list? Windows is the only thing with a platform-level comment, macOS and Linux are grouped above but Windows is down here, CopyFlutterBundle is in the middle of the Android stuff despite not being (at least based on the name) Android-specific, etc. If it's at the point (which I agree that it is) where a platform grouping comment is useful, maybe do a quick overhaul of the whole list to make it consistent for all platforms.

Reply:
I reorganized the list a bit. I agree its unnecessarily messy